methods.tex

```diff
@@ -313,14 +308,14 @@ \subsubsection{The SaltProc modeling and simulation code}
   \label{fig:saltproc_flow}
 \end{figure}
 SaltProc can manage as many material streams as desired. It also may work with 
-multiple depletion materials. At the end of a each depletion step, SaltProc 
+multiple depletion materials. At the end of each depletion step, SaltProc 
 reads the depleted compositions and tracks each material stream individually. 
 Following this, it applies chemical separation functions to fuel stream 
 vectors. These vectors then form a matrix (isotopics x timesteps) which 
 SaltProc stores in an HDF5 database and prints into the SERPENT 2 composition 
 file for the next depletion calculation.
 
-Saltproc datasets are timseries, meaning that every value is recorded every 
+Saltproc datasets are time-series, meaning that every value is recorded every 
```

Comment:
I don't think I like hyphenating time series. Perhaps "SaltProc records every value every timestep. The resulting time series datasets produced by SaltProc are listed below.... "

Reply:
see https://github.com/andrewryh/msbr-reproc/pull/3